### ISSUE TITLE
ONYX-719: Ruby stage crashing

### ISSUE THREAD

**Issue by ymarcon** [obiba/onyx#739]
Jira issue originally created by user mboulang:

1) Complete the CI questionnaire
2) Start the blood/urine stage

Unexpected RuntimeException

WicketMessage: Exception in rendering component: [Component id = 1]

Root cause:

java.lang.NullPointerException
at org.obiba.onyx.ruby.core.domain.ConditionalMessage.getArguments(ConditionalMessage.java:62)
at org.springframework.context.support.AbstractMessageSource.getMessage(AbstractMessageSource.java:147)
at org.springframework.context.support.DelegatingMessageSource.getMessage(DelegatingMessageSource.java:71)
at org.springframework.context.support.AbstractApplicationContext.getMessage(AbstractApplicationContext.java:987)
at org.obiba.wicket.model.MessageSourceResolvableStringModel.getObject(MessageSourceResolvableStringModel.java:70)
at org.apache.wicket.Component.getModelObject(Component.java:1558)
at org.apache.wicket.Component.getModelObjectAsString(Component.java:1580)
at org.apache.wicket.markup.html.basic.Label.onComponentTagBody(Label.java:111)
at org.apache.wicket.Component.renderComponent(Component.java:2480)
at org.apache.wicket.markup.html.WebComponent.onRender(WebComponent.java:60)
at org.apache.wicket.Component.render(Component.java:2317)
at org.apache.wicket.markup.repeater.AbstractRepeater.renderChild(AbstractRepeater.java:119)
at org.apache.wicket.markup.repeater.AbstractRepeater.onRender(AbstractRepeater.java:100)
at org.apache.wicket.Component.render(Component.java:2317)
at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:639)
at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:112)
at org.apache.wicket.Component.renderComponent(Component.java:2480)
at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
at org.apache.wicket.Component.render(Component.java:2317)
at org.apache.wicket.Component.renderComponent(Component.java:2420)
at org.apache.wicket.ajax.AjaxRequestTarget.respondComponent(AjaxRequestTarget.java:785)
at org.apache.wicket.ajax.AjaxRequestTarget.respondComponents(AjaxRequestTarget.java:662)
at org.apache.wicket.ajax.AjaxRequestTarget.respond(AjaxRequestTarget.java:577)
at org.apache.wicket.request.AbstractRequestCycleProcessor.respond(AbstractRequestCycleProcessor.java:104)
at org.apache.wicket.RequestCycle.processEventsAndRespond(RequestCycle.java:1181)
at org.apache.wicket.RequestCycle.step(RequestCycle.java:1252)
at org.apache.wicket.RequestCycle.steps(RequestCycle.java:1353)
at org.apache.wicket.RequestCycle.request(RequestCycle.java:493)
at org.apache.wicket.protocol.http.WicketFilter.doGet(WicketFilter.java:355)
at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:200)
at org.mortbay.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1148)
at org.springframework.orm.hibernate3.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:198)
at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
at org.mortbay.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1148)
at org.mortbay.jetty.servlet.ServletHandler.handle(ServletHandler.java:387)
at org.mortbay.jetty.security.SecurityHandler.handle(SecurityHandler.java:216)
at org.mortbay.jetty.servlet.SessionHandler.handle(SessionHandler.java:181)
at org.mortbay.jetty.handler.ContextHandler.handle(ContextHandler.java:765)
at org.mortbay.jetty.webapp.WebAppContext.handle(WebAppContext.java:417)
at org.mortbay.jetty.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:230)
at org.mortbay.jetty.handler.HandlerCollection.handle(HandlerCollection.java:114)
at org.mortbay.jetty.handler.HandlerWrapper.handle(HandlerWrapper.java:152)
at org.mortbay.jetty.Server.handle(Server.java:326)
at org.mortbay.jetty.HttpConnection.handleRequest(HttpConnection.java:534)
at org.mortbay.jetty.HttpConnection$RequestHandler.content(HttpConnection.java:879)
at org.mortbay.jetty.HttpParser.parseNext(HttpParser.java:747)
at org.mortbay.jetty.HttpParser.parseAvailable(HttpParser.java:218)
at org.mortbay.jetty.HttpConnection.handle(HttpConnection.java:404)
at org.mortbay.io.nio.SelectChannelEndPoint.run(SelectChannelEndPoint.java:409)
at org.mortbay.thread.QueuedThreadPool$PoolThread.run(QueuedThreadPool.java:520)

Complete stack:

org.apache.wicket.WicketRuntimeException: Exception in rendering component: [Component id = 1]
at org.apache.wicket.Component.renderComponent(Component.java:2519)
at org.apache.wicket.markup.html.WebComponent.onRender(WebComponent.java:60)
at org.apache.wicket.Component.render(Component.java:2317)
at org.apache.wicket.markup.repeater.AbstractRepeater.renderChild(AbstractRepeater.java:119)
at org.apache.wicket.markup.repeater.AbstractRepeater.onRender(AbstractRepeater.java:100)
at org.apache.wicket.Component.render(Component.java:2317)
at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1297)
at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1476)
at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:639)
at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:112)
at org.apache.wicket.Component.renderComponent(Component.java:2480)
at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1411)
at org.apache.wicket.Component.render(Component.java:2317)
at org.apache.wicket.Component.renderComponent(Component.java:2420)
at org.apache.wicket.ajax.AjaxRequestTarget.respondComponent(AjaxRequestTarget.java:785)
at org.apache.wicket.ajax.AjaxRequestTarget.respondComponents(AjaxRequestTarget.java:662)
at org.apache.wicket.ajax.AjaxRequestTarget.respond(AjaxRequestTarget.java:577)
at org.apache.wicket.request.AbstractRequestCycleProcessor.respond(AbstractRequestCycleProcessor.java:104)
at org.apache.wicket.RequestCycle.processEventsAndRespond(RequestCycle.java:1181)
at org.apache.wicket.RequestCycle.step(RequestCycle.java:1252)
at org.apache.wicket.RequestCycle.steps(RequestCycle.java:1353)
at org.apache.wicket.RequestCycle.request(RequestCycle.java:493)
at org.apache.wicket.protocol.http.WicketFilter.doGet(WicketFilter.java:355)
